clusters/build-clusters/common/hack/generate-bw-items.sh

```diff
@@ -11,8 +11,11 @@ set -o pipefail
 WORKDIR="$(mktemp -d)"
 readonly WORKDIR
 
-CONTEXT="build02"
-readonly CONTEXT
+declare -a App_CI_SAArray=("config-updater" "deck" "plank" "sinker" "hook" "crier" "release-bot" "prow-controller-manager" "pj-rehearse")
+declare -a Build01_SAArray=("config-updater" "deck" "plank" "sinker" "hook" "crier" "dptp-controller-manager" "prow-controller-manager" "ci-operator")
+declare -a Build02_SAArray=("${Build01_SAArray[@]}")
+
+declare -a Context_Array=("app.ci" "build01" "build02")
```

Comment:
vsphere as well or not?

Reply:
I kind of do not want to do vsphere with this script (at least not directly).
Because i do not usually put the kubeconfig into the default file.
I am not an admin there. I am using the one for `config-updater` only for testing on the current story.

I will see how often we have to login to vsphere. Maybe we should do the right thing: group ci-admins to vphere.